templates/vue/src/components/tyde/activities/CircleOfSupport/CircleView/index.vue

```diff
@@ -5,15 +5,19 @@
       class="tab"
       :connections="connections"
       :communities="communities"
+      :toolTipPositioned="toolTipPositioned"
       :draggable="!dragDisabled"
       @back="handleBack"
-      @add-connection="$emit('add-connection', $event)"
+      @add-connection="handleConnectionOpen"
+      @connection-opened="handleConnectionOpened"
       @edit-connection="handleEditConnection"
       @add-community="$emit('add-community', $event)"
       @delete-connection="$emit('delete-connection', $event)"
       @drag:start="handleDragStart"
       @drag:move="handleDragMove"
       @drag:end="handleDragEnd"
+      @connection-submitted="$emit('connection-submitted')"
+      @connection-closed="handleConnectionClosed"
```

Comment:
Can we put all these event listeners beside each other? It's a bit messy to see some down here and the rest up there ^